core/src/components/action-sheet/test/basic/index.html

```diff
@@ -62,7 +62,7 @@
         --ion-safe-area-bottom: 40px;
 
         --max-width: 300px;
-        --height: 325px;
+        --height: calc(325px + var(--ion-safe-area-bottom));
```

Comment:
This avoid unnecessary diffs with our basic test, except for places we are explicitly setting the `--ion-safe-area-bottom` to a custom value (action-sheet-custom-diff screenshots).